readme.md

```diff
@@ -2,4 +2,123 @@ Omnipay Tatra module
 ====================
 
 
-Work in progress
+# **ComfortPay**
+
+##### Charge
+
+Method to create transaction with registered card.
+
+Request:
+    
+    ChargeRequest
+
+Parameters:
+
+    - transactionId (required) - Unique ID of transaction
```

Comment:
Please wrap name of the parameters to with backticks, so they are displayed `like_this`.